onnxruntime/core/providers/cpu/nn/layer_norm_impl.h

```diff
@@ -65,6 +65,7 @@ class LayerNormImpl : public OpKernel {
   const bool contrib_op_;
   mutable IAllocatorUniquePtr<float> scale_fp32_;
```

Comment:
Please avoid mutable member variables for thread-safe.

Reply:
It seems that they are introduced to store the result of `MlasConvertHalfToFloatBuffer` so that it will not be called every time.

Reply:
@amarin16 @fajin-corp do you think it is OK to remove those 2 member fields?

Reply:
The reason I added the scale and bias as class members is because they can be populated from `PrePack` when the tensors are constant and then used inside methods called by `Compute`. The reason they are mutable is because `OpKernel.Compute` virtual method was const before these were added

Reply:
I see. Then we can leave them as it is. 
@fe-eire, could you add some comments there.